docker/mountebank/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM mhart/alpine-node
 EXPOSE 2525
 ENTRYPOINT ["mb"]
 CMD ["start","--mock"]
-ENV MOUNTEBANK_VERSION=1.9.0
+ENV MOUNTEBANK_VERSION=2.2.0
```

Comment:
I also needed to update to the newest version of mountebank to get the acceptance tests to pass.

Reply:
Thanks. I don't generally use docker when testing so I'm not surprised this stuff was out of date.